photonpump/conversations.py

```diff
@@ -299,6 +301,42 @@ def __init__(
         self.result.set_result(result)
 
 
+class ReadAllEventsBehaviour:
+    def __init__(self, result_type, response_cls):
```

Comment:
There's a couple of bugs I can see here - this class is still trying to use the value "self.stream" on lines 320 and 336.

I think we can just drop this class and make the ReadStreamEventsBehaviour understand when it's reading `$all`.